ansible/roles/rke/tasks/main.yml

```diff
@@ -0,0 +1,132 @@
+---
+- import_tasks: ssl-certificates.yml
+  run_once: true
+
+- name: install apt packages
+  become: true
+  apt:
+    name:
+      - unzip
+      - tar
+  when: ansible_distribution == 'Ubuntu'
```

Comment:
Do these come with rhel / centos?

Reply:
These come installed with the AMI's we use in AWS for e2e testing. It's possible some AMI's don't have them, but we won't really know until we get into the field.